Source/Core/DolphinTool/ConvertCommand.cpp

```diff
@@ -286,9 +286,8 @@ int ConvertCommand(const std::vector<std::string>& args)
         return EXIT_FAILURE;
       }
 
-      const std::pair<int, int> range =
-          DiscIO::GetAllowedCompressionLevels(compression_o.value(), false);
-      if (compression_level_o.value() < range.first || compression_level_o.value() > range.second)
+      const auto [fst, snd] = DiscIO::GetAllowedCompressionLevels(compression_o.value(), false);
+      if (compression_level_o.value() < fst || compression_level_o.value() > snd)
```

Comment:
> I'm guessing this is "first" and "second"?

Yes, those are the names the IDE suggested.

Reply:
`fst` and `snd` are not appropriate variable names. Not only are there clearer names you can use in many cases, but even if you have to name them `first` and `second` because there are no other good options, you should name them `first` and `second` rather than the overly abbreviated `fst` and `snd`. My immediate impression of what `fst` and `snd` would mean in the context of Dolphin are "file system table" and "sound" respectively, which shows how ambiguous these kinds of abbreviations can be.

Reply:
I'm not a fan of them either, but I thought it was some standard since they were suggested. I'll adopt each of your suggestions instead.

Reply:
No offense, but if you haven't even looked at the code you're refactoring to see if the variable names make sense, *why* are you PRing this in the first place?

Reply:
I meant I thought it was a coding standard for either C++ or coding in general and I was trying to follow it even though I don't agree with it. I've seen enough `idx` and single-letter variables to know people like to shorten variable names whenever possible.